e2e/tests/loopRequest.test.ts

```diff
@@ -12,17 +12,31 @@ describe('loop request', () => {
 
   it('rmf responds to loop request', () => {
     $('[data-component=MainMenu] [data-item=Robots]').click();
-    const robotItem = $('[data-component=RobotItem]');
-    robotItem.click();
-    const position = robotItem.$('[data-role=position]').getText();
-    robotItem.$('button=Loop').click();
-    robotItem.$('input[name=numLoops]').setValue(1);
-    robotItem.$('button=Request').click();
-    robotItem.$('button=Info').click();
-    expect(robotItem.$('[data-role=position]')).not.toHaveText(position);
-  });
 
-  it('renders robot trajectory', () => {
-    expect($('[data-component=RobotTrajectory]')).toBeVisible();
+    browser.addLocatorStrategy('findAllRobots', () => {
+      return document.querySelectorAll('[data-component=RobotItem]');
+    });
+
+    let getRobotLocations = () => {
+      const allRobotItems = browser.custom$$('findAllRobots', '[data-component=RobotItem]');
+      let robotLocations = allRobotItems.map(robot => {
+        robot.click();
+        return robot.$('[data-role=position]').getHTML();
+      });
+      return robotLocations;
+    };
+    const currentRobotLocations = getRobotLocations();
+
+    const backButton = $('[name="back-button"]');
+    backButton.click();
+    $('[data-component=MainMenu] [data-item=Commands]').click();
+    $('input[name=numLoops]').setValue(1);
+    $('button=Request').click();
+
+    backButton.click();
+    $('[data-component=MainMenu] [data-item=Robots]').click();
+    const newRobotLocations = getRobotLocations();
+
+    expect(newRobotLocations).not.toMatchObject(currentRobotLocations);
```

Comment:
I'm getting this error on the e2e test. It's opening the robot accordion instead of the command one.

```
ERROR webdriver: Request failed with status 400 due to element not interactable: element not interactable
  (Session info: chrome=83.0.4103.116)
[0-0] element not interactable in "loop request rmf responds to loop request"
element not interactable
  (Session info: chrome=83.0.4103.116)
```

Reply:
I got a similar error as well, in my case, it seems like the loop request is in an accordion but the test is not updated to click on it.

Reply:
Okay let me get to this, my oversight!!